docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,47 @@
+| [Exploratory Testing](guides/finding-game-breaking-bugs-with-stability-testing) | Run more than just predefined scenarios - exploratory bots such as the Monkey Bot can explore edge cases within your game. |
+| [Composability](core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments) | One approach is not always adequate for automating a certain test scenario. Our drag and drop tool allows you to stitch together and re-use bot logic across tests. |
+| [Runtime, CI/CD, and Editor Compatibility](core-concepts/running-tests-in-the-unity-test-runner) | Our tools work in the Editor, in builds, and within CI/CD. Because of our integration approach, in most cases you will **not need to restart your game** to iterate on a test scenario. |
+| [Data Exploration and Validation](core-concepts/validation-suites) | Our dashboard allows developers to inspect and test the state of their game in a no-code fashion. |
 
-This product is currently a **release preview**. We are actively talking with studios and game developers to discover the best features and use cases to tackle, and we're making changes frequently.
+## Where do I start?
 
-The team has focused on the foundation of supporting use cases and initial infrastructure to get bots to connect to Unity. The bulk
-of our value will come from the interfaces to build bots, the scale of our infrastructure, the speed of integration, and the products
-built around these bots.
+The best place to start is with our [installation guide](getting-started/installing-regression-games). Once our SDK is added to your project, you can [create your first
+test automation](getting-started/creating-your-first-automated-test).
 
-![timeline feature](unused-content/session-insights/img/replay/timeline.png)
+If you are already familiar with the framework, check out these more advanced guides for test inspiration:
 
-## Availability Assurances
+* [Interacting with User Interfaces using Computer Vision](guides/interacting-with-ui-using-computer-vision)
+* [Finding Game-Breaking Bugs with Stability Testing](guides/finding-game-breaking-bugs-with-stability-testing)
+* [Writing Advanced Validations](guides/writing-advanced-validations)
+* [Running Tests in Editor and in Builds](guides/running-tests-in-editor-and-in-builds)
 
-The Regression Games team offers the following assurances for our initial testing with developers. Additionally, we have a dedicated Discord server for support, and can also create a Slack Connect channel, giving you direct access to our entire team. For access to
-these channels, send an e-mail to [aaron@regression.gg](mailto:aaron@regression.gg).
+You can also check out our [existing game samples](additional-game-samples) or the [live demo of our validation tool](https://play.regression.gg/validations-demo).
```

Comment:
I'd move this higher up, possibly even before the installation link. If someone is coming to this doc to figure out whether they want to give RG a try then showing them the live demo is high impact with minimal effort on their part. Not sure what the additional-game-samples page is supposed to contain at this point, so can't say for that link.

Reply:
Moved around! But also open to changing this a bit.